axelor-base/src/main/java/com/axelor/apps/base/web/LanguageController.java

```diff
@@ -0,0 +1,16 @@
+package com.axelor.apps.base.web;
+
+import com.axelor.apps.base.db.Language;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+
+public class LanguageController {
+  public void convertToLowercase(ActionRequest request, ActionResponse response) {
+    Language language = request.getContext().asType(Language.class);
+    if (language.getCode() != null) {
+      response.setValue("code", language.getCode().toLowerCase());
+    } else {
+      response.setError("The Code field must not be empty!");
+    }
```

Comment:
Don't put this error message, the field is already required in the view and in database